pkg/commands/git.go

```diff
@@ -211,6 +195,21 @@ func NewGitCommandAux(
 // this returns the root of the current worktree. So if you start lazygit from within
 // a subdirectory of the worktree, it will start in the context of the root of that worktree
 func findWorktreeRoot(fs afero.Fs, currentPath string) (string, error) {
```

Comment:
We can actually remove this function because it's dead code now (same with its associated test)